check_tags.sh

```diff
@@ -33,6 +33,25 @@ if [ "$GITHUB_TOKEN" != "" ]; then
       if [[ "$SHOULD_BUILD" != "yes" ]]; then
         echo "Already have all the Linux ia32 builds"
       fi
+    elif [[ $BUILDARCH == "arm64" ]]; then
+      HAVE_ARM64_RPM=$(echo $VSCODIUM_ASSETS | jq 'map(.name) | contains(["arm64.rpm"])')
```

Comment:
Since MS vscode has a bug in their code for arm64 RPM, I think we should not check/build arm64 rpm's at all until it is fixed (otherwise the entire arm64 build process will run on every pass because it will find the rpm package missing). So this line and the if block L40-43 should be commented out for now.